server/package.json

```diff
@@ -13,6 +13,8 @@
     "axios": "^0.19.0",
     "bcrypt": "^3.0.6",
     "body-parser": "^1.19.0",
+    "dotenv": "^8.0.0",
+    "esm": "^3.2.25",
     "express": "^4.17.1",
```

Comment:
The command `nodemon -r esm -r dotenv/config index.js -` will work like this:
1. require ESM (for using ES6 format in our server application)
2. require dotenv/config (for loading environment variables in the application)
3. Then start the index.js file

Note: Work of -r is preloading the specified module at startup.